CrashReport 3.1.3: RuntimeException: Invalid night mode 48

**Describe the bug**
Night Mode returned an invalid value

**Expected behavior**
Map invalid value to a state, not an exception

**Environment**
Device Details:
Device:[karnak]
Model:[KFKAWI]
Manufacturer:[Amazon]
Android Version:[7.1.2]

**Logs**
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.kiwix.kiwixcustomphet/org.kiwix.kiwixmobile.custom.main.CustomMainActivity}: java.lang.RuntimeException: Invalid night mode 48
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2681)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2748)
at android.app.ActivityThread.-wrap12(ActivityThread.java)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1488)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:154)
at android.app.ActivityThread.main(ActivityThread.java:6188)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:911)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:801)
Caused by: java.lang.RuntimeException: Invalid night mode 48
at org.kiwix.kiwixmobile.core.NightModeConfig$UiMode$Companion.from(NightModeConfig.kt:4)
at org.kiwix.kiwixmobile.core.NightModeConfig.isNightModeActive(NightModeConfig.kt:4)
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.updateNightMode(CoreMainActivity.java:1)
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.loadPrefs(CoreMainActivity.java:22)
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.selectTab(CoreMainActivity.java:10)
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.newTab(CoreMainActivity.java:3)
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.newMainPageTab(CoreMainActivity.java:4)
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.getCurrentWebView(CoreMainActivity.java:1)
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.loadPrefs(CoreMainActivity.java:17)
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.onCreate(CoreMainActivity.java:81)
at org.kiwix.kiwixmobile.custom.main.CustomMainActivity.onCreate(CustomMainActivity.kt:1)
at android.app.Activity.performCreate(Activity.java:6689)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2634)
... 9 more

--- a/core/src/main/java/org/kiwix/kiwixmobile/core/NightModeConfig.kt
+++ b/core/src/main/java/org/kiwix/kiwixmobile/core/NightModeConfig.kt
@@ -61,7 +61,8 @@ class NightModeConfig @Inject constructor(
   enum class UiMode(val value: Int) {
     ON(Configuration.UI_MODE_NIGHT_YES),
     OFF(Configuration.UI_MODE_NIGHT_NO),
-    NOT_SET(Configuration.UI_MODE_NIGHT_UNDEFINED);
+    NOT_SET(Configuration.UI_MODE_NIGHT_UNDEFINED),
+    UNKNOWN(Configuration.UI_MODE_NIGHT_MASK); // Value returned from amazon devices
 
     companion object {
       @JvmStatic

